src/components/ArticlesListPage/ArticlesListPage.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { Container, Box, Typography, Button } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import useSearchQuery from '../../hooks/useSearchQuery';
+import SortButton from '../SortButton/SortButton';
+import SearchBar from '../SearchBar/SearchBar';
+import { ArticlesList } from '../ArticlesList/ArticlesList';
```

Comment:
Both `SearchBar` and `ArticlesList` files export just one thing. However, one uses named exports, and the other uses a default export. Avoid that.